relm4-book/src/factory_advanced.md

```diff
@@ -2,19 +2,19 @@
 
 > If you're not familiar with the `Rc` type of the standard library, have a look at [this](https://doc.rust-lang.org/std/rc/index.html).
 
-The `FactoryVec` we used in the previous chapter is sufficient for simple applications where elements only need to be added and removed from the back. Yet a common use case would be to add elements before another one or to remove a specific element. That introduces more complexity that needs to be taken care of but fortunately is mostly handled by Relm4.
+The `FactoryVec` we used in the previous chapter is sufficient for simple applications where elements only need to be added and removed from the back. Yet a common use case would be to add elements before another one or to remove a specific element. That introduces additional complexity that needs to be taken care of but fortunately this is mostly handled by Relm4.
 
-To show this, we'll create a similar counter app to the one of the previous chapter, but on **steroids**: we'll add functionality to add counters before and after a specific counter and to remove a certain counter. To get the flexibility we need for this, we'll use the `FactoryVecDeque` type instead of a `FactoryVec`.
+To show this, we'll create a similar counter app to the one of the previous chapter, but this time on **steroids**: we'll add functionality to add counters before and after a specific counter and to remove a certain counter. To get this flexibility we need for this, we'll use the `FactoryVecDeque` type instead of a `FactoryVec`.
```

Comment:
I think "To get ___the___ flexibility we need for this," is still better because otherwise "this" would be used twice.

Reply:
I suggested the --> this as _the_ seems rather unspecific. Maybe we could also say: To get the _required_ flexibility.

Reply:
"To get the required flexibility" sounds good to me :+1: